library/Ivoz/Provider/Domain/Model/CallCsvScheduler/CallCsvSchedulerDto.php

```diff
@@ -4,6 +4,9 @@
 
 class CallCsvSchedulerDto extends CallCsvSchedulerDtoAbstract
```

Comment:
Would be possible to use CompanyInterface Type contants in this file instead of all existing strings for vpbx, residential, retail and so on?